extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -462,9 +462,9 @@ public function getStatsData($startYear, $startMonth, $startDay, $endYear, $endM
 					$data[$row['reviewer_id']] = array(
 						'name' => $user->getName(),
 						'total' => 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "total" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)